terraform/data_template.tf

```diff
@@ -172,3 +175,22 @@ resource "template_dir" "grafana_config" {
     prometheus_url = "http://${var.prometheus_ip}:9090"
   }
 }
+
+# Keycloak
+data "template_file" "keycloak_install" {
+  template = file("../scripts/install_keycloak.sh.tpl")
+
+}
+
+# defile config-file path de for copying to EC2 machine from local
+# resource "template_dir" "keycloak_config" {
+#   source_dir      = "../configs/keycloak/"
+#   destination_dir = "../configs/keycloak/conf.render/"
+
+#   # vars = {
+#   #   db_endpoint = module.keycloak.db_endpoint
```

Comment:
Separate DB is not use in this case?

Reply:
sẽ cập nhật DB riêng trong release version 2. Hiện tại, vẫn đang cài đặt trên DB mặc định của source